.github/workflows/build-release-artifacts.yml

```diff
@@ -0,0 +1,105 @@
+name: Build Release Artifacts
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  release-artifacts:
+    runs-on: ubuntu-latest
+      
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up environment
+      uses: ./.github/actions/setup-rel-env
+    
+    - name: Install krew
+      run: | 
+        make krew
+        echo "${KREW_ROOT:-$HOME/.krew}/bin" >> $GITHUB_PATH
+      
+    - name: Build CRD yaml
+      run: make generate manifests
+    
+    - name: Lint the charts
+      run: make lint
+
+    - name: Build vdb-gen
+      run: | 
+        mkdir -p bin
+        make vdb-gen
+        pwd
+        ls -lhrt bin/
+
+    - name: Build RBAC yaml
+      run: |
+        mkdir -p config/samples/rbac
+        make create-default-rbac
+        ls -lhrt config/samples/rbac/
+
+    - name: Build Bundle
+      run: | 
+        make bundle
+        ls -lhrt
+
+    - name: Build helm charts
+      run: | 
+        make create-helm-charts
+        cd helm-charts
+        helm package verticadb-operator
+        ls -lhrt
+    
+    - name: Upload CRD
+      uses: actions/upload-artifact@v2
+      if: always()
+      continue-on-error: false
+      with:
+        name: release-artifacts
+        path: /home/runner/work/vertica-kubernetes/vertica-kubernetes/helm-charts/verticadb-operator/crds/verticadbs.vertica.com-crd.yaml 
+
+    - name: Upload Custom SCC
+      uses: actions/upload-artifact@v2
+      if: always()
+      continue-on-error: false
+      with:
+        name: release-artifacts
+        path: /home/runner/work/vertica-kubernetes/vertica-kubernetes/config/samples/custom-scc.yaml
+    
+    - name: Upload RBAC
+      uses: actions/upload-artifact@v2
+      if: always()
+      continue-on-error: false
+      with:
+        name: release-artifacts
+        path: /home/runner/work/vertica-kubernetes/vertica-kubernetes/config/samples/rbac/default-rbac.yaml
+
+    - name: Upload Helm Charts
+      uses: actions/upload-artifact@v2
+      if: always()
+      continue-on-error: false
+      with:
+        name: release-artifacts
+        path: /home/runner/work/vertica-kubernetes/vertica-kubernetes/helm-charts/*.tgz
+
+    - name: Upload vdb-gen
+      uses: actions/upload-artifact@v2
+      if: always()
+      continue-on-error: false
+      with:
+        name: release-artifacts
+        path: /home/runner/work/vertica-kubernetes/vertica-kubernetes/bin/vdb-gen
+
+    - name: Upload Bundle
+      uses: actions/upload-artifact@v2
+      if: always()
+      continue-on-error: false
+      with:
+        name: release-artifacts
+        path: /home/runner/work/vertica-kubernetes/vertica-kubernetes/bundle
```

Comment:
I noticed in your runs of this workflow that it doesn't include the bundle directory.  Just the subdirectories in it.  Since the bundle is not used when creating the GitHub release, only when updating the operator in operatorhub, maybe add this to a different artifact called olm-bundle?  Maybe call the other artifact `github-release-assets` (to align with the name they use when creating the GitHub release)?

Reply:
Sounds good.